terminus-ui/pipes/src/truncate/truncate.pipe.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  Pipe,
+  PipeTransform,
+} from '@angular/core';
+import { isNumber } from '@terminus/ngx-tools';
+
+
+
+/**
+ * Define the accepted string values for the {@link TsTruncateAt} position
```

Comment:
These links need to be the full class name `TsTruncateAtPipe` (same for line 19)